app/views/drafts/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row row-content">
+  <section class="action">
+    <div class="cta">
+      <%#= link_to 'Publish Draft', publish_collection_draft_path(get_resource), method: :post, class: 'eui-btn--blue spinner' %>
+      <%= link_to 'Delete Draft', "#delete-draft-modal", class: 'display-modal' %>
```

Comment:
Should probably be `Delete Variable Draft` -- consider https://stackoverflow.com/questions/13547144/i18n-for-model-specific-rails-submit-button, IF it would work. I know this is a modal so it would likely need to be specified but for forms and related buttons we could/should use the method described in that answer.